src/content/language/metadata.md

```diff
@@ -16,10 +16,27 @@ annotation begins with the character `@`, followed by either a reference
 to a compile-time constant (such as `deprecated`) or a call to a
 constant constructor.
 
-Four annotations are available to all Dart code: 
-[`@Deprecated`][], [`@deprecated`][], [`@override`][], and [`@pragma`][]. 
-For examples of using `@override`,
-see [Extending a class][].
+Metadata can appear before a library, class, typedef, type parameter,
+constructor, factory, function, field, parameter, or variable
+declaration and before an import or export directive.
+
+## Built-in annotations
+
+The following annotations are available to all Dart code: 
+
+*   [`@awaitNotRequired`][]: Suppress `unawaited_futures` and
```

Comment:
I haven't reviewed this whole thing, but wanted to leave a quick note:

`@awaitNotRequired` isn't available to all Dart code. It's part of [`package:meta`](https://pub.dev/packages/meta) similar to many [other annotations](https://pub.dev/documentation/meta/latest/meta/) (like `@visibleForTesting`).

If you want to document it and other `package:meta` annotations, consider adding a new section (maybe something like "Analyzer supported annotations") to parallel with this new section.

Reply:
Thanks, Parker! In this case, I'm going remove this PR and just have us follow what we were doing before. So maybe just a note in the 3.9 blog if that's how we've been doing things.